### PR TITLE
Issue/3228 jupyter redirect text

### PR DESCRIPTION
**Pull request by e-carlin**
This branch is based on issue/3229-privacy-notice. Please review and merge PR #3231  before reviewing this.

### CODE REVIEW

File: sirepo/package_data/static/html/login.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="row">
     <div class="col-sm-6 col-sm-offset-3"><div class="text-center">
       <a style="margin: 0 10px" ng-repeat="m in login.authService.methods" data-ng-href="{{ m.url }}" class="btn btn-primary">{{ m.label }}</a>
+      <p class="help-block">By signing up for Sirepo you agree to Sirepo’s <a href="https://www.sirepo.com/en/privacy.html">privacy policy</a> and <a href="https://www.sirepo.com/en/terms.html">terms and conditions</a>, and to receive informational and marketing communications from RadiaSoft. You may unsubscribe at any time.</p>
```
Comment (robnagler) [inline sirepo/package_data/static/html/login.html:5]: no absolute URLs just "/en/privacy.html"

File: sirepo/package_data/static/js/sirepo-components.js [hunk 2]
```diff
@@ -2593,19 +2597,27 @@ SIREPO.app.directive('emailLogin', function(requestSender, errorService) {
         restrict: 'A',
         scope: {},
         template: [
-            '<div class="row text-center">',
-              '<p>Enter your email address and we\'ll send an authorization link to your inbox.</p>',
+            '<div data-ng-show="isJupyterhub" class="alert alert-info col-sm-offset-2 col-sm-10" role="alert">',
+            'We’re improving your Jupyter experience by making both Jupyter and Sirepo accessible via a single email login. Simply follow the directions below to complete this process.',
```
Comment (robnagler) [inline sirepo/package_data/static/js/sirepo-components.js:2601]: apostrophe is confusing vs single quote. rather use backslash single quote for `We\'re`.

File: sirepo/package_data/static/js/sirepo-components.js [hunk 3]
```diff
@@ -2593,19 +2597,27 @@ SIREPO.app.directive('emailLogin', function(requestSender, errorService) {
...
                   '<input type="text" class="form-control" data-ng-model="data.email" required/>',
                   '<div class="sr-input-warning" data-ng-show="showWarning">{{ warningText }}</div>',
                 '</div>',
               '</div>',
-              '<div class="row text-center" style="margin-top: 10px">',
-                 '<button data-ng-click="login()" class="btn btn-primary">Continue</button>',
+              '<div class="form-group">',
+                '<div class="col-sm-offset-2 col-sm-10">',
+                  '<button data-ng-click="login()" class="btn btn-primary">Continue</button>',
+                  '<p class="help-block">By signing up for Sirepo you agree to Sirepo’s <a href="https://www.sirepo.com/en/privacy.html">privacy policy</a> and <a href="https://www.sirepo.com/en/terms.html">terms and conditions</a>, and to receive informational and marketing communications from RadiaSoft. You may unsubscribe at any time.</p>',
```
Comment (robnagler) [inline sirepo/package_data/static/js/sirepo-components.js:2619]: apostrophe vs single quote and no absolute uris.

**Review by robnagler — CHANGES_REQUESTED** [review]

File: sirepo/package_data/static/js/sirepo-components.js (hunk 2 above)
Comment (e-carlin) [reply sirepo/package_data/static/js/sirepo-components.js]: 8f6dbc4f

File: sirepo/package_data/static/js/sirepo-components.js (hunk 3 above)
Comment (e-carlin) [reply sirepo/package_data/static/js/sirepo-components.js]: https://github.com/radiasoft/sirepo/pull/3232/commits/b152a032d3e8d5bea02db98e32c6c8f53223ad8c

File: sirepo/package_data/static/html/login.html (hunk 1 above)
Comment (e-carlin) [reply sirepo/package_data/static/html/login.html]: https://github.com/radiasoft/sirepo/pull/3232/commits/b152a032d3e8d5bea02db98e32c6c8f53223ad8c

**Comment by e-carlin** [comment]
@robnagler all comments addressed. Review and merge #3231  before looking at this one. This branch is based on the branch for #3231. If you look in both places you will have to make 2x as many comments:)

**Pull request merged**